pkg/apis/metal/v1alpha1/types_infrastructure.go

```diff
@@ -15,10 +15,35 @@ type InfrastructureConfig struct {
 }
 
 type Firewall struct {
-	Size       string            `json:"size"`
-	Image      string            `json:"image"`
-	Networks   []string          `json:"networks"`
-	RateLimits map[string]uint32 `json:"rateLimits"`
+	Size            string           `json:"size"`
+	Image           string           `json:"image"`
+	Networks        []string         `json:"networks"`
+	MachineNetworks []MachineNetwork `json:"machinenetworks"`
```

Comment:
Does the user really need to specify these complex machine networks at shoot creation time? I thought this was something dynamic?

Reply:
That's right - removed it.